lib/internal/Magento/Framework/GraphQl/Query/ResolverInterface.php

```diff
@@ -25,13 +24,13 @@
      * @param array|null $value
      * @param array|null $args
      * @throws \Exception
-     * @return Value
+     * @return mixed
```

Comment:
Can we make the return type to be `Value|mixed`?

Reply:
Fixed